src/main/java/com/github/onsdigital/babbage/template/handlebars/helpers/SignedURLHelper.java

```diff
@@ -0,0 +1,71 @@
+package com.github.onsdigital.babbage.template.handlebars.helpers;
+
+import com.auth0.jwt.JWT;
+import com.auth0.jwt.algorithms.Algorithm;
+import com.auth0.jwt.exceptions.JWTCreationException;
+import com.github.jknack.handlebars.Handlebars;
+import com.github.jknack.handlebars.Options;
+import com.github.onsdigital.babbage.configuration.Configuration;
+import com.github.onsdigital.babbage.template.handlebars.helpers.base.BabbageHandlebarsHelper;
+
+import java.io.IOException;
+import java.util.Arrays;
+
+/**
+ * Created by iankent on 03/02/17.
+ */
+public enum SignedURLHelper implements BabbageHandlebarsHelper<String> {
+
+    //Sign a URL
+    sign_url {
```

Comment:
Enum should be all upper case.

Reply:
don't think we can for these, they're actually template functions and it'd be inconsistent with all the existing helpers

Reply:
fine.